src/ui/server/controllers/insurance/business/contact/index.test.ts

```diff
@@ -153,7 +153,7 @@ describe('controllers/insurance/business/contact', () => {
 
     describe('when there are no validation errors', () => {
       it('should redirect to next page', async () => {
-        req.body = mockBusinessContact;
+        req.body = mockContact;
```

Comment:
Maybe `mockPolicyContact` instead of `mockContact`?

Reply:
can this be changed after I remove business contact?